wiki/Announcements/Archive/en.md

```diff
@@ -10,170 +10,215 @@ osu!weekly is a weekly news digest covering the latest happenings of the game an
-| [osu!weekly #13](//osu.ppy.sh/news/120821096528)                                         | The Catch the Beat World Cup finals begin this weekend! Last week the teams for...                           |
-| [osu!weekly #12](//osu.ppy.sh/news/120228961713)                                         | The Catch the Beat World Cup semifinals start this weekend! Last week the teams for...                       |
-| [osu!weekly #11](//osu.ppy.sh/news/119646062918)                                         | The Catch the Beat World Cup quarterfinals start this weekend! If you were watching last...                  |
-| [osu!weekly #10](//osu.ppy.sh/news/119064807443)                                         | The Catch the Beat World Cup is going into the Round of 16 this weekend!...                                  |
-| [osu!weekly #9](//osu.ppy.sh/news/118494468743)                                          | The Catch the Beat World Cup group stage is starting this weekend! The first match...                        |
-| [osu!weekly #8](//osu.ppy.sh/news/117886541118)                                          | Applications for the Beatmap Nominator Group are ending on Saturday! If you have been a...                   |
-| [osu!weekly #7](//osu.ppy.sh/news/117304714888)                                          | Applications for Beatmap Nominator Group are now open! If you have been a contributing, helpful...           |
-| [osu!weekly #6](//osu.ppy.sh/news/116706490333)                                          | The Taiko World Cup 2015 has concluded with Japan taking first place against Taiwan in...                    |
-| [osu!weekly #5](//osu.ppy.sh/news/116084773988)                                          | The Taiko World Cup grand finals are going to be underway this upcoming Sunday. Last...                      |
-| [osu!weekly #4](//osu.ppy.sh/news/115437619683)                                          | Get your catchers ready boys and girls, as the osu! Catch the Beat World Cup...                              |
-| [osu!weekly #3](//osu.ppy.sh/news/114799715018)                                          | Beatmap Nomination Group applications are open just a little while longer, closing March 29’th. If...        |
-| [osu!weekly #2](//osu.ppy.sh/news/114180316308)                                          | Hello again! I’m Tasha, and I’m here to bring you the second week of osu!weekly...                           |
-| [osu!weekly #1](//osu.ppy.sh/news/113518228483)                                          | Hello there! I’m Tasha, and I’m here to bring you the first week of osu!weekly!                              |
-
+You can email any tips for news directly to `tips@ppy.sh` or `news@ppy.sh`.
+
+- [osu!weekly #99](https://osu.ppy.sh/news/159271999848)
+- [osu!weekly #98](https://osu.ppy.sh/news/158993380768)
```

Comment:
Some snippets were inconsistent on where I had taken the snippet (It usually is the first paragraph after the osu!weekly image)... also, this may require translators to translate that part only when the osu!weekly is in English (or am I wrong here?)